dace/sdfg/sdfg.py

```diff
@@ -2517,7 +2517,10 @@ def simplify(self, validate=True, validate_all=False, verbose=False, options=Non
             :note: This is an in-place operation on the SDFG.
         """
         from dace.transformation.passes.simplify import SimplifyPass
-        return SimplifyPass(validate=validate, validate_all=validate_all, verbose=verbose,
+        return SimplifyPass(validate=validate,
+                            validate_all=validate_all,
+                            verbose=verbose,
+                            skip=options.get('skip', None),
```

Comment:
you'll have to make sure `options` is not None first.

Reply:
Why doesn't `SDFG.simplify` have a skip argument? It would be nicer for a user.

Reply:
that's probably even better, yes - we should just propagate the argument up to the next API level

Reply:
I agree, creating a skip argument is better IMO too.